contour/install.hbs.md

```diff
@@ -92,40 +91,79 @@ To install Contour from the Tanzu Application Platform package repository withou
+                                                                                                       on an Openshift cluster, this must be set to openshift. When set to openshift,    
+                                                                                                       a Role and RoleBinding are created to associate Contour's controllers with the    
+                                                                                                       appropriate Openshift Security Context Constraint resource.                       
+          kubernetes_version                                        0.0.0                     string   The version of Kubernetes being used, for enabling version-specific behaviors.    
+                                                                                                       Accept any valid major.minor.patch version of Kubernetes. This field is           
+                                                                                                       optional. Currently only has effect when kubernetes_distribution is set to        
+                                                                                                       openshift.                                                                        
+          namespace                                                 tanzu-system-ingress      string   The namespace in which to deploy Contour and Envoy.                               
+          registry_secret_names                                     [contour-reg-creds]       array    The names of the placeholder secrets that will contain registry credentials to    
+                                                                                                       pull the Contour and Envoy images.                                                
+          certificates.duration                                     8760h                     string   How long the certificates should be valid for.                                    
+          certificates.renewBefore                                  360h                      string   How long before expiration the certificates should be renewed.                    
+          contour.pspNames                                          vmware-system-restricted  string   Pod security policy names to apply to Contour.                                    
+          contour.replicas                                          2                         integer  How many Contour pod replicas to have.                                            
+          contour.resources.contour.requests.memory                                           string   Memory request to apply to the contour container.                                 
+          contour.resources.contour.requests.cpu                                              string   CPU request to apply to the contour container.                                    
```

Comment:
I wonder if we should make a note in the release notes about being able to set resource limits and requests for the Contour and Envoy pods now?

Reply:
release notes added. Let me know what you think

Reply:
nice, looks good to me